### PR TITLE
Trinary: Test for 4, as invalid trinary value

### PR DESCRIPTION
**Pull request by kotp**
Solutions that check for decimals in "normalization" should be stricter
than simply checking for decimals.

### CODE REVIEW

**Comment by Insti** [comment]
Looks fine.

Probably unrelated issue: Shouldn't Trinary.new('4') raise an error rather than something that allows to_decimal to be called on it?

Update: Ok I've read the README, I still think it should raise an error though.

File: exercises/trinary/trinary_test.rb [hunk 1]
```diff
@@ -52,4 +52,28 @@ def test_invalid_trinary_with_digits_is_decimal_0
     skip
```
Comment (Insti) [inline exercises/trinary/trinary_test.rb]: Another extra thing: 
Should this function be called: `test_invalid_trinary_with_invalid_digits_is_decimal_0`

Comment (kotp) [inline exercises/trinary/trinary_test.rb]: Definitely an incomplete title.

**Comment by kotp** [comment]
I think it is consistent behavior that '4' results in 0 as well.

* Readme
* Other tests, i.e., 'carrot' -> 0, '0a1b2c' -> 0
* Ruby's own default coercion: i.e., `String#to_i`, `String#to_f`, etc.

**Comment by Insti** [comment]
I agree it's consistent.

**Comment by kotp** [comment]
Brought in @Insti change for the multiline strings, as it is appropriate.  /cc @exercism/xruby

File: exercises/trinary/trinary_test.rb [hunk 2]
```diff
@@ -52,4 +52,33 @@ def test_invalid_trinary_with_digits_is_decimal_0
...
+  #  In your file, it will look like this:
+  #
+  # module BookKeeping
+  #   VERSION = 1 # Where the version number matches the one in the test.
+  # end
+  #
+  # If you are curious, read more about constants on RubyDoc:
+  # http://ruby-doc.org/docs/ruby-doc-bundle/UsersGuide/rg/constants.htm
+  def test_bookkeeping
+    assert_equal 1, BookKeeping::VERSION
```
Comment (Insti) [inline exercises/trinary/trinary_test.rb]: Would version 2 be more appropriate? Or is there a convention that the unversioned version is always 0?

Comment (kotp) [inline exercises/trinary/trinary_test.rb]: I have been using the no version indicated is implied 0, and once we start keeping track, it becomes version 1.

**Comment by Cohen-Carlisle** [comment]
:+1:

**Pull request merged**